spec/routing/flash_cards_routing_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+describe FlashCardsController do
+  it do
+    expect(get: '/').to route_to(
+      controller: 'flash_cards',
+      action: 'index')
+  end
+
+  it do
+    expect(get: '/').not_to route_to(
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.